requirements/build.txt

```diff
@@ -1,6 +1,72 @@
-pip==9.0.1 --hash=sha256:690b762c0a8460c303c089d5d0be034fb15a5ea2b75bdf565f40421f542fefb0
-setuptools==36.2.7 --hash=sha256:02b5f3869777dfbfbe3786a057bc6d523445a3e541a0abdf85adbe0b091e96b9
-wheel==0.29.0 --hash=sha256:ea8033fc9905804e652f75474d33410a07404c1a78dd3c949a66863bd1050ebd
-numpy==1.13.1 --hash=sha256:d910a24f536f926bd56fb30d6f17ae8b89a1406e105087a49e014e000b00e8db
-pycparser==2.18 --hash=sha256:99a8ca03e29851d96616ad0404b4aad7d9ee16f25c9f9708a11faf2810f7b226
-cffi==1.10.0 --hash=sha256:e7175287f7fe7b1cc203bb958b17db40abd732690c1e18e700f10e0843a58598
+pip==9.0.1 \
+    --hash=sha256:09f243e1a7b461f654c26a725fa373211bb7ff17a9300058b205c61658ca940d \
+    --hash=sha256:690b762c0a8460c303c089d5d0be034fb15a5ea2b75bdf565f40421f542fefb0
```

Comment:
Don't we pick up `pip` from the Docker image? Do we need to install it, too?

Reply:
Yep, I think this is a candidate for removal as well. There was a big jump in ``pip`` versions, they are up to 19.2.1.